docs/NotebookInstructions.md

```diff
@@ -5,55 +5,38 @@ This example notebook demonstrates how to use the [Dataset Insights](https://git
 
 ## Workflow
 
-* If you do not have Conda installed on your computer, install Conda. We recommend [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
+* Make sure [Docker Desktop](https://www.docker.com/products/docker-desktop) is installed.
 
-* Once Conda is installed: 
-  * On Mac OS, open a new terminal window.
-  * On Windows, you will need to open either Anaconda Prompt or Anaconda Powershell Prompt. These can be found in the Start menu.
+* Open a command line and use the following command to download and run the Dataset Insights Docker image, and mount your local folders to it:
 
-* Create a virtual environment named `synthdet_env` using Conda, and activate it:
-
-```bash
-conda create -n synthdet_env python=3.7
-conda activate synthdet_env
 ```
-
-* Install Dataset Insights:
-
-```bash
-pip install datasetinsights
+docker run -p 8888:8888 -v <datasets_path>:/data -v <synthdet_notebook_path>:/tmp -t unitytechnologies/datasetinsights:latest
 ```
 
-* Install Jupyter:
-
-```bash
-conda install jupyter
-```
+In the above command, `<datasets_path>` is the path to the folder where all locally generated SynthDet datasets are stored, and `<synthdet_notebook_path>` is the location at which the `SynthDet_Statistics.ipynb` notebook file is located inside your SynthDet repository (`repository_root/Notebooks`). 
 
-* You now need to create a kernel for your virtual environment, and add your virtual environment to Jupyter:
 
-```bash
-conda install -c anaconda ipykernel
-python -m ipykernel install --user --name=synthdet_env
+On OSX, `<datasets_path>` is:
 ```
+/Users/username/Library/Application\ Support/UnityTechnologies/SynthDet/
+```
+And on Windows, it is:
+```
+C:\Users\username\AppData\LocalLow\UnityTechnologies\SynthDet
+```
+> :information_source: Remember to replace `username` in the above paths.
 
-* Navigate to the folder where the `SynthDet_Statistics.ipynb` notebook file is located.
-  
-* You now can open the notebook using the command:
+> :information_source: If using a Linux/Unix based OS, make sure the spaces in both paths are escaped with backslashes, as shown in the above example.
 
-```bash
-jupyter notebook
-```
+> :information_source: If you get an error about the format of the command, try the command again **with quotation marks** around the folder mapping arguments, i.e. `"<datasets_path>:/data"`.
```

Comment:
Might as well use quotation marks in the example, to remove 2/3 infos. It's standard practice and would allow spaces to not be escaped (I think?)

Reply:
can confirm it works with quotations and non escaped paths